docs/components/add-another.md

```diff
@@ -38,6 +38,20 @@ This component has been tested in prototypes of several citizen and internal pro
 
 ## Accessibility issues
 
-Some people who use assistive technology will find it hard to identify form items that are added. This is because the form labels this component creates are given the same ID when added to the page. [A fix for this has been proposed on GitHub.](https://github.com/ministryofjustice/moj-frontend/issues/160) This has been raised in an external audit under Web Content Accessibility Guidelines (WCAG) 2.4.6 Headings and Labels (Level AA). If you use this component without addressing this issue, you must list it in the accessibility statement.
+### Identifying form items
 
-People with low vision have also reported difficulty interacting with the remove button. This is because by default it is aligned to the right of the page away from their main focus.
+Some people who use assistive technology will find it hard to identify new form items (for example ‘first name’) that are created when they use the ‘add another’ button. This is because the form labels all have the same ID.
```

Comment:
> This is because the form labels all have the same ID

This isn't quite correct - the labels don't have IDs, the issue is that all the labels have the same label text. So all the form inputs have the same label text, making it hard to distinguish between them.